start-site/src/main/resources/application.yml

```diff
@@ -429,7 +429,7 @@ initializr:
           description: Okta specific configuration for Spring Security/Spring Boot OAuth2 features. Enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.
           groupId: com.okta.spring
           artifactId: okta-spring-boot-starter
-          compatibilityRange: "[2.1.2.RELEASE,2.4.0-M1)"
+          compatibilityRange: "2.4.0"
```

Comment:
@snicoll is this the correct format to set the minimum version, or _should_ it be something like `[2.4.0,)`?

Reply:
I am not keen to have open ranges for 3rd party libraries so I'll polish that to be 2.4.x so that the entry is disabled with `2.5.x`. As always if you have a CI build that works with that generation just ping us and we'll relax the range.

Reply:
That said You've just removed support for 2.2.x and 2.3.x. Why do you want to remove the minimum version to be 2.4.x as we have previous okta versions that work fine with those Spring Boot generations?

Can you please review [the documentation](https://docs.spring.io/initializr/docs/current/reference/html/#dependencies-compatibility-range)?

Reply:
Super helpful! thank you!